### DISCUSSION TITLE
Batch change colour profiles using UNC path

### DISCUSSION THREAD

**Discussion by pgm-norbert** [ImageMagick/ImageMagick#5753] [Help]
Hi Everyone,

I'm trying to run this bat file using UNC path, but I can't get it working. Is there a workaround to use this, or it will only work if I have my drive mapped? 

`cd //Volumes/Shared Folder/`
`for /R %%f in (*.jpg) do magick "%%f" -profile C:\Users\default\Desktop\sRGB.icc "%%f"`
`for /R %%f in (*.jpeg) do magick "%%f" -profile C:\Users\default\Desktop\sRGB.icc "%%f"`
`cd..`

Any advice would be much appreciated.

**Comment by snibgo** [comment]
> ...  but I can't get it working.

Some more information may give a clue. Does the `cd` work? Any error messages? How do you know it doesn't work?

**Reply by pgm-norbert** [reply]
Apologies...
No. `cd` doesn't work to start with. 
I'm running it from Windows 10. I tried the portable version as well, and no joy. 
Is this the limitation of cmd not supporting UNC paths?

**Reply by pgm-norbert** [reply]
Just sorted it. 
Instead of `cd` and `cd ..` I've used `pushd` and `popd`
More info can be found here: 
[https://superuser.com/questions/282963/browse-an-unc-path-using-windows-cmd-without-mapping-it-to-a-network-drive](url)
